### PR TITLE
Pass error msg from quay.io to user

### PR DESCRIPTION
**Pull request by MartinBasti**
Don't hide reason why push failed.

Signed-off-by: Martin Bašti <mbasti@redhat.com>

### CODE REVIEW

File: operatorcourier/push.py [hunk 1]
```diff
@@ -44,4 +44,13 @@ def _push_to_registry(self, namespace, repository, release, bundle, auth_token):
         r = requests.post(push_uri, json=json, headers=headers)
         if r.status_code != 200:
             logger.error(r.text)
-            raise ValueError("Registry api returned non 200 response. Failed to push to app registry.")
+
+            msg = 'Failed to get error details'
+            try:
+                error = r.json()['error']
+                msg = "Quay.io answered: '{}' ({})".format(
+                    error['code'], error['message'])
```
Comment (twaugh) [inline operatorcourier/push.py]: Are these the right way round?

Comment (MartinBasti) [reply operatorcourier/push.py]: IMO yes, do you thing that `Quay.io answered: long message ('shortcode')` is better? (I have no strong preference)

Comment (twaugh) [reply operatorcourier/push.py]: I have no strong preference either, just thought you probably wanted the code within the parentheses. :)

Comment (MartinBasti) [reply operatorcourier/push.py]: I'd leave it as is then

**Pull request merged**